utils/db_utils.py

```diff
@@ -69,16 +69,59 @@ def query_confirmed_trips(start_date: str, end_date: str, tz: str):
     ts = esta.TimeSeries.get_aggregate_time_series()
     # Note to self, allow end_ts to also be null in the timequery
     # we can then remove the start_time, end_time logic
-    entries = ts.find_entries(
-        key_list=["analysis/confirmed_trip"],
+    df = ts.get_data_df("analysis/confirmed_trip",
         time_query=estt.TimeQuery("data.start_ts", start_ts, end_ts),
     )
-    df = pd.json_normalize(list(entries))
+    user_input_cols = []
 
-    # logging.debug("Before filtering, df columns are %s" % df.columns)
+    logging.debug("Before filtering, df columns are %s" % df.columns)
     if not df.empty:
-        columns = [col for col in perm_utils.get_all_trip_columns() if col in df.columns]
+        # Since we use `get_data_df` instead of `pd.json_normalize`,
+        # we lose the "data" prefix on the fields and they are only flattened one level
+        # Here, we restore the prefix for the VALID_TRIP_COLS from constants.py
+        # for backwards compatibility. We do this for all columns EXCEPT:
+        # 1. the coordinates, which we will have to pull out from the geojson anyway
+        # 2. the user_id, which doesn't need to be copied
+        # 3. the primary modes, which have not yet been populated
```

Comment:
```suggestion
        # for backwards compatibility. We do this for all columns since columns which don't exist are ignored by the rename command.
```